docs/DeveloperGuide.md

```diff
@@ -283,18 +283,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BookFace` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 1: Delete a book record**
```

Comment:
A better labelling to the use case could be:
`**Use case: UC01 - Delete a book record`